src/main/resources/db.changelog/changes/office.yaml

```diff
@@ -0,0 +1,81 @@
+databaseChangeLog:
+  - logicalFilePath: no-path
+  - changeSet:
+      id: office teble init
+      author: saidaminqambarov408@gmail.com
+      preConditions:
+        onFail: MARK_RAN
+        not:
+          tableExists:
+            schemaName:
+            tableName: office
+      changes:
+        - createTable:
+            tableName: office
+            columns:
+              - column:
+                  name: of_id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    primaryKeyName: pk_of_id
+              - column:
+                  name: ad_id
+                  type: bigint
+                  constraints:
+                    unique: true
+                    foreignKeyName: fk_ad_id
+                    references: address(ad_id)
+              - column:
+                  name: pa_id
+                  type: bigint
+                  constraints:
+                    unique: true
+                    nullable: true
+                    foreignKeyName: fk_pa_id
+                    references: parking(pa_id)
+              - column:
+                  name: of_status
+                  type: varchar(255)
+              - column:
+                  name: of_created
+                  type: datetime
+                  defaultValueComputed: CURRENT_TIMESTAMP
+              - column:
+                  name: of_modified
+                  type: datetime
+
+#   - changeSet:
+#       id: change ad_id column datatype
```

Comment:
Why it commented?

Reply:
I want to use this but it was not necessary